content/concepts/security-considerations.md

```diff
@@ -0,0 +1,140 @@
+---
+title: 'Security Considerations'
+weight: 12
+---
+
+libp2p makes it simple to establish [encrypted, authenticated communication
+channels](../secure-comms/) between two peers, but there are other important
+security issues to consider when building robust peer-to-peer systems.
+
+Many of the issues described here have no known "perfect solution," and the
+solutions and mitigation strategies that do exist may come with tradeoffs and
+compromises in other areas. As a general-purpose framework, libp2p tries to
+provide the tools for application developers to address these problems, rather
+than taking arbitrary approaches to security that may not be acceptable to all
+systems built with libp2p.
+
+
```

Comment:
Maybe add a note about the practicability of attacks?

> Another aspect to consider is that the fact that a particular type of attack is theoretically feasible, does not automatically imply that it is practical, sensible, worthwhile, or efficient to carry out. To evaluate the actual exploitability of a theoretical attack vector, consider the volume, class, and cost of resources that the attacker would have to expend for their attack to reasonably succeed.

Reply:
Very minor nit, but I would phrase as "to have a reasonable chance of succeeding."